lib/tasks/checkins.rake

```diff
@@ -48,4 +48,16 @@ namespace :checkins do
       end
     end
   end
+
+  desc 'Imports checkins from CSV'
+  task :import, [:filepath, :device_id] => :environment do |_t, args|
+    Checkin.transaction do
+      CSV.foreach(args[:filepath], headers: true) do |row|
+        checkin = Checkin.find_by_id(row['id']) || Checkin.new
```

Comment:
@t-harps Also, please note that import is a little bit different than export.

During import we have to deal with DB inserts, and so difference between many inserts (of 1 row) and one insert (of several rows) it's much more important than creating of several objects in Ruby.

This acticle explains it pretty well: https://blog.codeship.com/speeding-up-bulk-imports-in-rails/